src/lightning/app/core/flow.py

```diff
@@ -741,22 +740,6 @@ def configure_api(self):
         """
         raise NotImplementedError
 
-    def configure_plugins(self) -> Optional[List[Dict[str, Plugin]]]:
```

Comment:
Better not to have this hook for now.

Reply:
Yes, we can add back later if it makes sense